src/views/QuantumTP6m/index.jsx

```diff
@@ -19,27 +21,16 @@ const styles = {
 };
 
 class TP6M extends React.Component {
-  constructor(props) {
-    super(props);
-    const { location } = this.props;
-    const params = new URLSearchParams(location.search);
-
-    this.state = {
-      test: params.get('test') || 'loadtime',
-      platform: 'android-hw-g5-7-0-arm7-api-16',
-    };
-  }
-
   render() {
-    const { classes } = this.props;
-    const { test, platform } = this.state;
+    const { classes, navigation, test, platform } = this.props;
     const subtitle = frum(TP6_TESTS)
       .where({ id: test })
       .first().label;
 
     return (
       <div className={classes.body}>
         <DashboardPage key={subtitle} title="TP6 Mobile" subtitle={subtitle}>
+          {navigation}
```

Comment:
If `navigation` is included in every page, then perhaps it would make sense to add this line in `DashboardPage`.

Reply:
Still too early to say; these particular pages are a uniform series of charts, while the others are heterogeneous.